tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -27,35 +27,68 @@ using namespace TCLAP;
 using namespace realdds;
 
 
+#define NAME2SERVER( X )                                                                                               \
```

Comment:
Since it is only used in `get_supported_streams` (including variable names) it is better to #define and #undef it in the function scope

Reply:
Now you're reaching... ;)